src/jtag3.c

```diff
@@ -1613,6 +1635,9 @@ static int jtag3_parseextparms(const PROGRAMMER *pgm, const LISTID extparms) {
         msg_error("  -xvtarg                 Read on-board target supply voltage\n");
         msg_error("  -xvtarg=<arg>           Set on-board target supply voltage\n");
       }
+      if(str_starts(pgmid, "pickit4") || str_starts(pgmid, "snap")) {
+        msg_error("  -xmode=avr|pic          Set programmer to AVR or PIC mode\n");
```

Comment:
Might want to say Set programmer to AVR or PIC mode, *then exit*.

Reply:
Good idea!